libexec/image-handler.sh

```diff
@@ -80,10 +80,12 @@ case "$SINGULARITY_IMAGE" in
 
         eval_abort "$SINGULARITY_libexecdir/singularity/python/import.py"
 
+         
         for i in `cat "$SINGULARITY_CONTENTS"`; do
             name=`basename "$i"`
             message 1 "Exploding layer: $name\n"
             zcat "$i" | (cd "$SINGULARITY_ROOTFS"; tar --exclude=dev/* -xf -) || exit $?
+            chmod -R +w "$SINGULARITY_ROOTFS"
```

Comment:
@hra0031 let me know if this looks akin to your change - I removed the extra message to make it a little less verbose.